### PR TITLE
【paddle.distributed】 add Fleet/RoleMaker API doc

### PR DESCRIPTION
**Pull request by seiriosPlus**
加入FleetAPI中文文档
加入RoleMaker中文文档

### CODE REVIEW

**Comment by seiriosPlus** [comment]
image

File: doc/fluid/beginners_guide/coding_practice/save_load_variables_en.rst [hunk 1]
```diff
@@ -196,23 +196,23 @@ For the PServer to be loaded with parameters during training, for example:
...
-		exe.run(pserver_prog)
-	if training_role == "TRAINER":
-		main_program = t.get_trainer_program()
-				exe.run(main_program)
+    pserver_endpoints = "127.0.0.1:1001,127.0.0.1:1002"
+    trainers = 4
+    Training_role == "PSERVER"
+    current_endpoint = "127.0.0.1:1002"
+    config = fluid.DistributeTranspilerConfig()
+    t = fluid.DistributeTranspiler(config=config)
```
Comment (guru4elephant) [inline doc/fluid/beginners_guide/coding_practice/save_load_variables_en.rst:187]: why fluid.DistributeTranspiler here?

File: doc/paddle/api/paddle/distributed/fleet/Fleet_cn.rst [hunk 2]
```diff
@@ -6,62 +6,359 @@ Fleet
...
+
+    import paddle.distributed.fleet as fleet
+    fleet.init(is_collective=True)
+
+**代码示例3**
+
+.. code-block:: python
+
+    import paddle.distributed.fleet as fleet
+    role = fleet.PaddleCloudRoleMaker
```
Comment (guru4elephant) [inline doc/paddle/api/paddle/distributed/fleet/Fleet_cn.rst]: PaddleCloudRoleMaker()?

File: doc/fluid/beginners_guide/coding_practice/save_load_variables_en.rst (hunk 1 above)
Comment (seiriosPlus) [reply doc/fluid/beginners_guide/coding_practice/save_load_variables_en.rst:187]: deprecated doc. merged from develop

**Review by guru4elephant — APPROVED** [review]
LGTM

**Pull request merged**